Doc/c-api/structures.rst

```diff
@@ -320,6 +320,46 @@ definition with the same method name.
    members can be deleted.  (They are set to *NULL*).
 
 
+.. c:type:: PyGetSetDef
+
+   Structure to define property-like access for a type. See also description of
+   the :c:member:`PyTypeObject.tp_getset` slot.
+
+   +-------------+------------------+-----------------------------------+
+   | Field       | C Type           | Meaning                           |
+   +=============+==================+===================================+
+   | name        | const char \*    | attribute name                    |
```

Comment:
It is `char *` in 2.7.

Reply:
Good catch. I've seen that this caused the cherry-pick conflict in the other file, but thought that was only a documentation issue.

I should've checked the code.